### DISCUSSION TITLE
Without calling Poco::Net::initializeSSL() and Poco::Net::uninitializeSSL() my application is running on secure server or socket

### DISCUSSION THREAD

**Discussion by Nikku3695** [pocoproject/poco#4277] [General]
In my application i am able to connect on secure sockets without calling initializeSSL(),
whether this is correct way to use bcoz i m getting some resources leak every time when we are creating a socket and deleting it,
what's the use of Poco::Net::initializeSSL() or poco is itself calling it internally?

**Comment by CJO-20** [comment]
Would like some follow up on this as well, noticed the same behavior.
